manual/manual/Makefile

```diff
@@ -76,7 +76,7 @@ html: files
 	         -I ../tutorials -I ../../styles -I ../texstuff manual.hva \
 	         -e macros.tex ../manual.tex ; \
 	${HACHA} -tocter manual.html ; \
-
+	cat ../manual.css >> manual.css
```

Comment:
Ocamldoc is not involved here, this is a modification of  *hevea* default style.
Moreover, only people somehow using the compiler/manual internal `manual/tool/caml_tex2" could be impacted.